e2e/app/run.go

```diff
@@ -264,7 +264,7 @@ func toPortalValidators(validators map[*e2e.Node]int64) ([]bindings.Validator, e
 func logRPCs(ctx context.Context, def Definition) {
 	network, endpoints := externalNetwork(def)
 	for _, chain := range network.EVMChains() {
-		rpc, _ := endpoints.GetByNameOrID(chain.Name, chain.ID)
+		rpc, _ := endpoints.ByNameOrID(chain.Name, chain.ID)
```

Comment:
nit: don't ignore the error, might fail in future.

Reply:
it will just log an empty value which is fine in this case